alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/resources/templates/transformForm.html

```diff
@@ -0,0 +1,162 @@
+				<!-- PDF Renderer only -->
+				<tr><td><div style="text-align:right">page</div></td><td><input type="text" name="page" value="" /></td></tr>
+				<tr><td><div style="text-align:right">width</div></td><td><input type="text" name="width" value="" /></td></tr>
+				<tr><td><div style="text-align:right">height</div></td><td><input type="text" name="height" value="" /></td></tr>
+				<tr><td><div style="text-align:right">allowPdfEnlargement</div></td><td><input type="checkbox" name="allowPdfEnlargement" value="true" /></td></tr>
+				<tr><td><div style="text-align:right">maintainPdfAspectRatio</div></td><td><input type="checkbox" name="maintainPdfAspectRatio" value="true" /></td></tr>
+				<!-- ImageMagick only -->
+				<tr><td><div style="text-align:right">startPage</div></td><td><input type="text" name="startPage" /></td></tr>
+				<tr><td><div style="text-align:right">endPage</div></td><td><input type="text" name="endPage" /></td></tr>
+				<tr><td><div style="text-align:right">alphaRemove</div></td><td><input type="text" name="alphaRemove" /></td></tr>
+				<tr><td><div style="text-align:right">autoOrient</div></td><td><input type="text" name="autoOrient" /></td></tr>
+				<tr><td><div style="text-align:right">cropGravity</div></td><td><input type="text" name="cropGravity" />North, NorthEast...Center</td></tr>
+                <tr><td><div style="text-align:right">cropWidth</div></td><td><input type="text" name="cropWidth" /></td></tr>
+				<tr><td><div style="text-align:right">cropHeight</div></td><td><input type="text" name="cropHeight" /></td></tr>
+				<tr><td><div style="text-align:right">cropPercentage</div></td><td><input type="checkbox" name="cropPercentage" value="true" /></td></tr>
+				<tr><td><div style="text-align:right">cropXOffset</div></td><td><input type="text" name="cropXOffset" /></td></tr>
+				<tr><td><div style="text-align:right">cropYOffset</div></td><td><input type="text" name="cropYOffset" /></td></tr>
+				<tr><td><div style="text-align:right">thumbnail</div></td><td><input type="checkbox" name="thumbnail" value="true" /></td></tr>
+				<tr><td><div style="text-align:right">resizeWidth</div></td><td><input type="text" name="resizeWidth" value="" /></td></tr>
+				<tr><td><div style="text-align:right">resizeHeight</div></td><td><input type="text" name="resizeHeight" value="" /></td></tr>
+				<tr><td><div style="text-align:right">resizePercentage</div></td><td><input type="checkbox" name="resizePercentage" value="true" /></td></tr>
+				<tr><td><div style="text-align:right">allowEnlargement</div></td><td><input type="checkbox" name="allowEnlargement" value="true" /></td></tr>
+				<tr><td><div style="text-align:right">maintainAspectRatio</div></td><td><input type="checkbox" name="maintainAspectRatio" value="true" /></td></tr>
+
+				<tr><td></td><td><input type="submit" value="Transform" /></td></tr>
+            </table>
+        </form>
```

Comment:
Do we need this form if we have the 5 separate ones below ?

Or else have form above with a named section for each set of sub-transformer specific options ?